operations/node_management/Configure_NTP_on_NCNs.md

```diff
@@ -75,9 +75,17 @@ The following steps are structured to be executed on one node at a time. However
 
 ## Fix broken configuration
 
+On each affected NCN run the following:
+
 1. Set a token as described in [Identify Nodes and Update Metadata](Rebuild_NCNs/Identify_Nodes_and_Update_Metadata.md).
 
-1. On each affected NCN, run:
+1. Export the token.
+
+    ```bash
+    ncn# export TOKEN
```

Comment:
should we show a sample value? or that a value is expected?  e.g.  {enter-your-token-value-here}